src/app/component/MissionCard.jsx

```diff
@@ -76,13 +90,72 @@ const MissionCard = withStyles(styles)(({ children, classes, mission, ...rest })
 
   return (
     <Card className={classes.root} {...rest}>
-      <CardHeader
-        title={title}
-        titleTypographyProps={{ variant: "h2", component: "span" }}
-        className={classes.cardHeader}
-      />
       <CardContent className={classes.cardContent}>
-        <MissionCardContent contentItems={contentItems} classes={classes} />
+        <Grid container spacing="1" alignItems="flex-start" justify="flex-end" direction="row">
+          <Grid item>
+            <img height="20" src={appleIcon} alt="" />
+          </Grid>
+          <Grid item style={{ flex: 1 }}>
+            {title}
+          </Grid>
+          <Grid item>
+            <InfoIcon />
```

Comment:
I assume from the design that this is supposed to take you to the same place as "View Details" does in this button currently? 

Cause in the latest design there is no "View Details" button - making space for "unassign me" and other things.

I think the idea is that the user can press anywhere in the title, or ideally almost anywhere in the bulk of the card.. to get to the details?